### ISSUE TITLE
[BUG] Failed to run the trigger on preview condition response

### ISSUE THREAD

**Issue by sihamais** [opensearch-project/alerting-dashboards-plugin#332]
**What is the bug?**

I am not able to preview trigger condition response when creating a bucket monitor from the dashboard.

**How can one reproduce the bug?**

1. Go to Alerting -> Monitors -> Create Monitor
2. Choose Per bucket monitor with an Extraction Query Editor.
3. Create a test bucket monitor with a simple aggregation. Here’s mine:
```json
{
    "size": 0,
    "query": {
        "match_all": {}
    },
    "aggregations": {
        "composite_agg": {
            "terms": {
                "field": "from.keyword"
            }
        }
    }
}
```
4. Create a simple trigger condition. I took the one from the documentation:
```json
{
  "buckets_path": {
    "count_var": "_count"
  },
  "parent_bucket_path": "composite_agg",
  "script": {
    "source": "params.count_var > 5"
  }
}
```
5. Click on the `Preview condition response` button under the trigger condition.
It displays this error: `[parsing_exception] Failed to parse object: expecting token of type [START_OBJECT] but found [END_OBJECT], with { line=1 & col=319 }`

**What is the expected behavior?**

It should display `true` or `false` according to the result of the condition without failing.

**What is your host/environment?**
OpenSearch on-premise v2.2.0
OpenSearch Dashboard v2.2.0

**Do you have any screenshots?**
![opensearch_trigger_preview_bug](https://user-images.githubusercontent.com/70810208/191984128-65d0d34f-4ae4-4fe4-94d1-260cb773b653.png)

**Do you have any additional context?**
This issue seems to only happen on bucket monitors.

**Comment by JoniLaurila** [comment]
We have same issue with UI. (2.3 Opensearch version, AWS Managed). 

The trigger works (if rightly configured) but we can't test trigger in UI because we get this same error message as in shown in orriginal post. 

This only happens with "Per bucket monitors"

**Comment by amsiglan** [comment]
Fixed in #382

**Comment by sihamais** [comment]
Do you know when the fix will be released and on what version? Thanks
